core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -12,9 +12,16 @@
 
 require 'solidus_core'
 
+# @private
+def forgery_protected_by_default?
+  Gem::Version.new(Rails.version) >= Gem::Version.new('5.2')
```

Comment:
Is this still needed with the latest master?

Reply:
Yep, still needed: https://github.com/solidusio/solidus/blob/master/core/solidus_core.gemspec#L27